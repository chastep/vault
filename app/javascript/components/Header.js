import React from 'react';
import { Header as HeaderUI, Icon } from "semantic-ui-react";

const Header = () => (
  <HeaderUI as='h1' textAlign='center' icon>
    <Icon name='lock' />
    Vault
    <HeaderUI.Subheader>
      Manage your Bank Accounts with ease!
    </HeaderUI.Subheader>
  </HeaderUI>
);

export default Header;
