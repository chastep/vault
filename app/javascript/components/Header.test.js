import React from 'react';
import renderer from 'react-test-renderer';

import Header from './Header';

it('renders', () => {
  const component = renderer.create(
    <Header />
  ).toJSON();
  expect.assertions(1);
  expect(component).toMatchSnapshot();
});
