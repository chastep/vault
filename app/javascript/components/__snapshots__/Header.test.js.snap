// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<h1
  className="ui icon center aligned header"
>
  <i
    aria-hidden="true"
    className="lock icon"
    onClick={[Function]}
  />
  Vault
  <div
    className="sub header"
  >
    Manage your Bank Accounts with ease!
  </div>
</h1>
`;
